src/ui/issuepanel/IssuePanel.java

```diff
@@ -38,6 +40,11 @@
 	private final KeyCombination defaultSizeWindow = new KeyCodeCombination(KeyCode.D, KeyCombination.CONTROL_DOWN);
 	private HashMap<Integer, Integer> issueCommentCounts = new HashMap<>();
 
+	private static final int BROWSER_REQUEST_DELAY = 500; //milliseconds
+	private static TickingTimer timer;
+	private static int nextIssueId;
+	private static int nextColumnIndex;
+
```

Comment:
NavigableListView defines the currently-selected issue index and ColumnControl the panel index. Probably better that we reuse that state instead of keeping track of it in multiple places.